pandas/tests/series/test_period.py

```diff
@@ -164,3 +164,13 @@ def test_end_time_timevalues(self, input_vals):
         result = s.dt.end_time
         expected = s.apply(lambda x: x.end_time)
         tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize('input_vals, expected', [
+        (Series(['2001'], dtype='datetime64[ns]'),
+         Series(['2001'], dtype='period[D]')),
+        (Series(['NaT'], dtype='datetime64[ns]'),
+         Series(['NaT'], dtype='period[D]'))
```

Comment:
Instead, just parameterize over `['2001'. 'NaT']`, then in the test body you can do:

```
expected = Series([arg], dtype='period[D]')
result = Series([arg], dtype='datetime64[ns]').dt.to_period('D')
```

Reply:
That makes a lot more sense. I was making the update late at night and wasn't thinking clearly. Thank you!